stbt-run

```diff
@@ -72,6 +80,35 @@ def _print_exc_utf8(file_=None):
         traceback._some_str = _old_some_str
 
 
+def _record_screenshot(exception):
```

Comment:
Might as well call this `_write_screenshot` (or `_save_screenshot`) to be consistent with the wording of the command-line argument.